README.md

```diff
@@ -1 +1,141 @@
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Portfolio<a name="about-project"></a>
+
+**Capstone-Project.1** is a project built Html, Css and Javascript.
+
+- Main task carried out include:
+
+Dynamically creating cards using Javascript DOM manipulation, organizing page layout using flex box, integrating media query at specific width and using custom backgrounds.
+- [About the project](https://www.loom.com/share/d5b26129cda04933a625c5ad0a2d03f7)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Test my current skills as a software developer**
+- **Showcases Languages, Frameworks, and Skills**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yin-ka.github.io/Capstone-Project.1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+### Usage
+
+To run the project open the home.html file in your web browser
+
+### Run tests
+
+No tests are available
+
+### Deployment
+
+You can deploy this project using GitHub Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+**Yin-ka**
+- GitHub: [https://github.com/yin-ka)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **The modal part of the portfolio will be added.**
+- [ ] **Links to project showcases will be active with JavaScript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse full-stack curriculum for the inspiration and guidance.
+
```

Comment:
- [ ] [OPTIONAL] Kindly make sure you acknowledged the owner of the original design in your README as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.) 👍